lib/jxl/encode_internal.h

```diff
@@ -117,12 +117,15 @@ struct JxlEncoderStruct {
   size_t num_queued_boxes;
   std::vector<jxl::JxlEncoderQueuedInput> input_queue;
   std::deque<uint8_t> output_byte_queue;
-  size_t output_bytes_flushed;
 
-  // Get the current write position in the stream (for indexing use).
-  size_t BytePosition() const {
-    return output_bytes_flushed + output_byte_queue.size();
-  }
+  // How many codestream bytes have been written, i.e.,
+  // content of jxlc and jxlj boxes. Frame index box jxli
```

Comment:
I think jxlj should be jxlp (also below), the codestream is jxlc and jxlp boxes

Reply:
fixed